src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -174,8 +173,6 @@ const modalScreenListeners = {
         Modal.setModalVisibility(false);
     },
     beforeRemove: () => {
-        // Clear search input (WorkspaceInvitePage) when modal is closed
-        SearchInputManager.searchInput = '';
```

Comment:
Shouldn't we clear it here?

Reply:
No, because we've removed the usage of persist on that page. Apparently navigation patterns have changed and this is no longer necessary.